chainer/datasets/svhn.py

```diff
@@ -32,11 +32,13 @@ def get_svhn(withlabel=True, scale=1., dtype=numpy.float32,
             the datasets only contain images.
         scale (float): Pixel value scale. If it is 1 (default), pixels are
             scaled to the interval ``[0, 1]``.
+        add_extra: Use extra training set.
         dtype: Data type of resulting image arrays.
         label_dtype: Data type of the labels.
 
     Returns:
-        A tuple of two datasets. If ``withlabel`` is ``True``, both datasets
+        If ``add_extra`` is ``False``, a tuple of two datasets. Otherwise,
```

Comment:
Please make it more clear. How about this?

```
If ``add_extra`` is ``False``, a tuple of two datasets (train and test). Otherwise,
a tuple of three datasets (train, test and extra).
```